test-d/set-mutable.ts

```diff
@@ -0,0 +1,18 @@
+import {expectType, expectError} from 'tsd';
```

Comment:
This file should be merged with the `mutable.ts` test.

Reply:
Yeah, I've done that but forgot to remove `set-mutable.ts`. Applied.